src/server/app_render.jsx

```diff
@@ -88,20 +75,15 @@ async function appRender(ctx) {
             userPreferences,
             offchain,
         });
-
-        // Assets name are found in `webpack-stats` file
-        const assets_filename =
-            ROOT +
-            (process.env.NODE_ENV === 'production'
-                ? '/tmp/webpack-stats-prod.json'
-                : '/tmp/webpack-stats-dev.json');
-        const assets = require(assets_filename);
-
-        // Don't cache assets name on dev
-        if (process.env.NODE_ENV === 'development') {
+        let assets;
+        if (!resolvedAssets) {
+            // Assets name are found in `webpack-stats` file
+            const assets_filename = ROOT + '/tmp/webpack-stats-dev.json';
```

Comment:
Don't we need to keep the `process.env.NODE_ENV === 'production'` switch?

Reply:
Oh, I see. `webpack-stats-prod.json` is resolved in `src/server/server.js` for `production`.

I guess the dev logic is here because it allows hot reloading?